ES/ESdd041.xml

```diff
@@ -177,144 +184,131 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                    
                      <handNote xml:id="h1" script="Ethiopic">
-                        <seg type="script">Fine and careful</seg>
-                        <seg type="ink">Black, red (strong red)</seg>
-                        <persName role="scribe" ref="PRS00000">Malkǝʾa Krǝstos</persName> is mentioned as the scribe in the colophon. His name is also
-                        written, in red, in the middle of the bottom margin on <locus target="#3r">3r</locus>.<date>Early 17th century</date>
+                        <seg type="script">Early 17th-century script.</seg>
+                        <seg type="ink">Black, red (strong red).</seg>
+                        <desc>Fine and carefull.
+                        <persName role="scribe" ref="PRS14616MalkeaKrestos">Malkǝʾa Krǝstos</persName> is mentioned as the scribe in the <ref target="#coloph1"/>. His name is also
+                           written, in red, in the middle of the bottom margin on <locus target="#3r"/>.</desc>
+                        
                         <list type="abbreviations">
                            <item>
-                              <abbr>ይ፡ ካ</abbr> for <expan>ይቤ፡ ካህን፡ </expan> (<locus target="#32vb">32vb</locus>, 33vb)</item>
+                              <abbr>ይ፡ ካ፡</abbr> for <expan>ይቤ፡ ካህን፡ </expan> (<locus target="#32vb"/>)</item>
                            <item>
-                              <abbr>ይ፡ ሕ፡</abbr> for <expan>ይቤ፡ ሕዝብ፡ </expan> (<locus target="#33rb">33rb</locus>, 33vb, 35ra)</item>
+                              <abbr>ይ፡ ሕ፡</abbr> for <expan>ይቤ፡ ሕዝብ፡ </expan> (<locus target="#33rb #33vb #35ra"/></item>
                         </list>
-                        <seg type="rubrication">Nomina sacra; names of the king, patriarch and
+                        <seg type="rubrication">Divine names; names of the king, patriarch and
                            metropolitan; names of the commissioner and scribe; a group of lines and
                            a few single lines (alternating with black lines) in the incipit page; a
                            word, a few words, a line or a few lines of the incipit of the sections
-                           and subsections; word ʾǝgale; names of the Hebrew letters (fols.
-                           12va-13rb); directives for the ministers; two initial lines of the
-                           colophon; elements of the punctuation signs; Ethiopic numerals or their
-                           elements.</seg>
+                           and subsections; word <foreign xml:lang="gez">ʾǝgale</foreign>; names of the Hebrew letters (<locus from="12va" to="13rb"/>); directives for the ministers; two initial lines of the
+                           colophon; elements of the punctuation signs; Ethiopic numerals or their elements.</seg>
                      </handNote>
                   </handDesc>
                   <decoDesc>
 
                      <decoNote type="frame" xml:id="d1">
```

Comment:
type is not "frame", it is "headpieceBand", in my opinion.

Reply:
good